ocs_ci/ocs/pgsql.py

```diff
@@ -366,21 +378,66 @@ def get_pgbech_pod_status_table(self, pgbench_pods):
         """
         pgbench_pod_table = PrettyTable()
         pgbench_pod_table.field_names = [
-            'pod_name', 'scaling_factor', 'num_clients', 'num_threads', 'trans_client',
-            'actually_trans', 'latency_avg', 'lat_stddev', 'tps_incl', 'tps_excl'
+            'pod_name', 'scaling_factor', 'num_clients', 'num_threads',
+            'trans_client', 'actually_trans', 'latency_avg', 'lat_stddev',
+            'tps_incl', 'tps_excl'
         ]
         for pgbench_pod in pgbench_pods:
             output = run_cmd(f'oc logs {pgbench_pod.name}')
             pg_output = utils.parse_pgsql_logs(output)
             for pod_output in pg_output:
                 for pod in pod_output.values():
-                    pgbench_pod_table.add_row([
-                        pgbench_pod.name, pod['scaling_factor'], pod['num_clients'], pod['num_threads'],
-                        pod['number_of_transactions_per_client'], pod['number_of_transactions_actually_processed'],
-                        pod['latency_avg'], pod['lat_stddev'], pod['tps_incl'], pod['tps_excl']
-                    ])
+                    pgbench_pod_table.add_row(
+                        [pgbench_pod.name, pod['scaling_factor'],
+                         pod['num_clients'], pod['num_threads'],
+                         pod['number_of_transactions_per_client'],
+                         pod['number_of_transactions_actually_processed'],
+                         pod['latency_avg'], pod['lat_stddev'],
+                         pod['tps_incl'], pod['tps_excl']]
+                    )
         log.info(f'\n{pgbench_pod_table}\n')
 
+    def export_pgoutput_to_googlesheet(self, pg_output, sheet_name, sheet_index):
+        """
+        Collect pgbench output to google spreadsheet
+
+        Args:
+            pg_output (list):  pgbench outputs in list
+            sheet_name (str): Name of the sheet
+            sheet_index (int): Index of sheet
+
+        """
+        # Collect data and export to Google doc spreadsheet
+        g_sheet = GoogleSpreadSheetAPI(
+            sheet_name=sheet_name, sheet_index=sheet_index
+        )
+        log.info("Exporting pgoutput data to google spreadsheet")
+        for i in range(len(pg_output)):
+            for j in range(len(pg_output[i][0])):
+                run_id = list(pg_output[i][0][j].keys())[0]
```

Comment:
Can use different variable names? not (i,j)

Reply:
Done